virt/virt-2-6-release-notes.adoc

```diff
@@ -60,10 +60,12 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-9455: Improvements to KubeVirt and CDI ensure that storage is allocated on the correct node
 
-//CNV-9456: When importing VM disk images, you can now control which multus network is used
+* When preparing local storage for a virtual machine disk image, a volume may be allocated from a different node than is required by the virtual machine. To prevent scheduling problems, this release integrates with the Kubernetes `WaitForFirstConsumer` binding mode to ensure that volumes are allocated from the correct node.
```

Comment:
s/may/might

It is not clear what "this release" means. Suggestion: `s/this release/{VirtProductName} now` or `s/this release/the Containerized Data Importer (CDI) now` depending on your context.

Reply:
Great, thanks.